corehq/motech/migrations/0001_initial.py

```diff
@@ -1,16 +1,17 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.11.12 on 2018-04-26 23:24
-from __future__ import unicode_literals
+# Generated by Django 1.11.12 on 2018-04-27 21:34
 from __future__ import absolute_import
+from __future__ import unicode_literals
 
-import jsonfield.fields
 from django.db import migrations, models
+import jsonfield.fields
 
 
 class Migration(migrations.Migration):
 
+    initial = True
```

Comment:
Does this make it work when the table already exists?

Reply:
Thanks for pointing this out. That tells Django to create the table ... except it wouldn't because `migrations.SeparateDatabaseAndState` prevents the table from being created. It's `migrations.SeparateDatabaseAndState` which makes it work when the table already exists.